opam

```diff
@@ -1,15 +1,21 @@
-opam-version: "1"
+opam-version: "1.2"
 maintainer: "dave.scott@citrix.com"
+authors: "Dave Scott"
+homepage: "https://github.com/mirage/xentropyd"
+bug-reports: "https://github.com/mirage/xentropyd/issues"
+license: "ISC"
 build: [
   [ "oasis" "setup" ]
   [ "ocaml" "setup.ml" "-configure" ]
   [ "ocaml" "setup.ml" "-build" ]
+  [ "cp" "main.native" "%{bin}%/xentropyd" ]
```

Comment:
Even though I did the update and merged it, I note that this looks wrong and should probably use a `xentropyd.install` file to make sure removal works properly or move the `cp` into `install` and add a `remove` with `rm -f`.